### DISCUSSION TITLE
3D Printer Model of Crazyfle 2.X 7mm Motor Mount

### DISCUSSION THREAD

**Discussion by benidiet** [bitcraze/discussions#1169] [General]
Hi,

would it be possible to get the 3d model of the crazyflie 2.x 7mm motor mount parts? I would like to extend them with the possibility to mount spring contact pins. 

Thanks and best regards
Benedikt

**Comment by tobbeanton** [comment]
Sorry but we can not share the molded part. There are some other version in the [mechanics repo](https://github.com/bitcraze/bitcraze-mechanics/tree/master/motor-mounts) you might be able to use though.
